lms/djangoapps/mobile_api/courses/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Courses API
+"""
```

Comment:
There must be a better location for this.  Why not implement this as part of the `friends` endpoint?  The `friends` endpoint can then support both searching by course (if given a course id) and retrieving all courses (if no course id is provided).  This would be consistent with standard API conventions.